client/packages/coldchain/src/Monitoring/api/TemperatureBreach/hooks/document/useTemperatureBreaches.ts

```diff
@@ -1,11 +1,19 @@
 import { useTemperatureBreachApi } from '../utils/useTemperatureBreachesApi';
-import { useQuery } from '@openmsupply-client/common';
+import { AuthError, LocalStorage, useQuery } from '@openmsupply-client/common';
 import { ListParams } from '../../api';
 
 export const useTemperatureBreaches = (queryParams: ListParams) => {
   const api = useTemperatureBreachApi();
 
-  return {
-    ...useQuery(api.keys.paramList(queryParams), api.get.list(queryParams)),
-  };
+  return useQuery(api.keys.paramList(queryParams), () =>
```

Comment:
this one though, still needs to tell the user that they do not have permission to view breaches